### DISCUSSION TITLE
Im very new, question whether it is working or not?

### DISCUSSION THREAD

**Discussion by Samuel-pixel-crypto** [Munkwolf/dcs-community-keybinds#68] [General]
Hey guys, brand new to Github and DCS here. I installed this for the ANT ELEV/ANT COLL function being too fast, hoping to get it to move slower. After doing my best attempt at what I believed was installing quaggles and this mod correctly, im not sure if it is actually working or not... did some light testing in game with the knob and it seems to be slower? just not sure because there are no new settings popping up and I just want to verify that it is actually working properly. Thanks!

**Comment by taipan303** [comment]
Hi,
F16? It should look like this:
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/12576248/95305a46-2102-47d2-85ca-c286b506a54a)

If you can't see the extra binds you may have the folders wrong.

Quaggles gets installed to the main DCS folders.
Munkwolf's gets installed to the saved games\dcs folders.

**Reply by Samuel-pixel-crypto** [reply]
yeah doesnt look like that unfortunately, I most likely did something wrong. Just to make sure, this works with open beta DCS right? Im not home at the moment but I can send pictures of what I attempted later today.

**Reply by Samuel-pixel-crypto** [reply]
I tried installing both on the OVGME mod manager, and both in the saved games DCS open beta folder. Which is the main DCS folder?

**Comment by taipan303** [comment]
Yeah most people only use open beta.

Probably just the folder structure is wrong on either one.

**Reply by Samuel-pixel-crypto** [reply]
how can I verify the way it should look? sorry for the trouble

**Comment by taipan303** [comment]
Quaggles installed to Main DCS folder for me looks like this path for standalone installed to D drive (but yours might be program files, or Steam):
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/12576248/c18f10c7-60eb-4250-ab23-3662ef31627c)

Munkwolf's command folders in saved games looks like this path:
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/12576248/7fc2bac8-adfa-4879-9f5c-15bb90edaf9c)

**Reply by Samuel-pixel-crypto** [reply]
should I disable OVGME then?

**Reply by Samuel-pixel-crypto** [reply]
I cannot find the DCS World OpenBeta folder, only have a DCS and DCS.openbeta. edit: found the folder, still doesnt show up...

**Reply by Samuel-pixel-crypto** [reply]
your file for quaggles seems to have much more content then mine? I only have a single file called "data" that is 86kbs

**Comment by taipan303** [comment]
Hi, that sounds like you put it in the wrong folder structure then. DCS has heaps of files in that structure.

**Reply by Samuel-pixel-crypto** [reply]
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/75598230/50055d97-4bb7-4a3e-a6b5-0169c2ac7cfa)

this is what it currently looks like, the one on the left I copy pasted Munkwolf's F16C edits into the default.lua

**Comment by taipan303** [comment]
We can't see the whole path on those screenshots, it cuts off before the DCS folder.
Also you shouldn't be editing any files, you only need to paste the folder structure into each one:
1. Quaggles input injector paste the matching folder structure into the main DCS install folder (DCS install folder means where the game was installed)
2. Munkwolfs commands paste into the saved games DCS folder structure (saved games is inside your user profile)

**Reply by Samuel-pixel-crypto** [reply]
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/75598230/1a63b4a2-ac29-4c6a-a29b-311ad1f6e598)

![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/75598230/a074c845-11b2-4b67-98d1-3eb86fb90b48)


my apologies, is this more useful?

**Reply by taipan303** [reply]
See updated comment above.

So the first folder with the inputcommands stucture looks like the correct place to put the Munkwolf community keybinds

Then second folder under eagle dynamics (DCS install folder) is incorrect - you should be putting only Quaggles input command injector in the main install folder, no keybind changes.

**Reply by taipan303** [reply]
ps going back to the screenshots of where mine is installed might be clearer now

**Reply by Samuel-pixel-crypto** [reply]
eh Ill try to make sense of it, very confusing to me. I have to just paste the munkwolf and not edit anything right?

**Reply by Samuel-pixel-crypto** [reply]
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/75598230/8549a0e1-619b-42ee-b814-dbc95230af20)

ok so I pasted Quaggles into the same file location (scripts) that your Screenshot shows, but it only has the single LUA file, is that normal?

edit: sorry if what Im doing is wrong and dumb, trying to understand

**Reply by Samuel-pixel-crypto** [reply]
also just to check, all of these folders have the empty templates in them, 
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/75598230/ea0e1abe-e0cb-4091-8298-9973dab8b3f0)

and I copy pasted the Munkwolf into the empty template for the "keyboard" into it right?
![image](https://github.com/Munkwolf/dcs-community-keybinds/assets/75598230/c7452eb7-0e38-4f6b-819d-aa84f3464dd7)


EDIT: ok so I just replaced all those template files with the munkwolf files

**Reply by Samuel-pixel-crypto** [reply]
HOLY FUCK I FIGURED IT OUT!! THANK YOU SO VERY VERY MUCH.

**Reply by Samuel-pixel-crypto** [reply]
I am deeply grateful, thank you so so so so very much.

**Comment by taipan303** [comment]
Awesome! Seems confusing at first, but worth it in the end

**Reply by Samuel-pixel-crypto** [reply]
one last thing, if the game receives an update, what do I need to do to make sure the mod/files still work?

**Reply by taipan303** [reply]
The Munkwolf files in saved games won't get touched by updates.

The main game files (Quaggles) can get overwritten by updates, you can use a mod manager to re apply Quaggles each update or do it manually.
